### PR TITLE
adds opennsfw model

### PR DESCRIPTION
**Pull request by scottleedavis**
Addresses Issue #103

### CODE REVIEW

File: store.json [hunk 1]
```diff
@@ -292,5 +292,12 @@
     "labels": {
       "com.openfaas.ui.ext": "jpg"
     }
+  },
+  {
+    "title": "Open NSFW Model",
+    "description": "Score images for NSFW (nudity) content.",
+    "image": "servernull/openfaas-opennsfw:0.1.0",
+    "name": "openfaas-opennsfw",
```
Comment (alexellis) [inline store.json]: Can we do without the prefix perhaps?

Comment (scottleedavis) [reply store.json]: Done

**Comment by scottleedavis** [comment]
I followed the rebase approach you asked for previously in Issue #102 , and will improve my approach to the process of contributing to openfaas.  I appreciate feedback always.

**Comment by alexellis** [comment]
For future reference please see the recommendations we make in the contributing guide for formatting commit messages and the body of the commit message. 

It's based upon this gem: https://chris.beams.io/posts/git-commit/

**Comment by alexellis** [comment]
Merged, thank you for your awesome function.

For a follow-up PR, how about setting a memory requests value?

Using the `metrics-server` app from `k3sup` or the helm chart you can monitor the memory used then send in a recommended value for the function. I did this for the inception function which is quite heavy-weight - around 1GB.

https://k3sup.dev/

**Comment by scottleedavis** [comment]
Hi @alexellis , I ran `metrics-server`, though I am not sure I captured it correctly it as the values are quite low.
```
NAME                                 CPU(cores)   MEMORY(bytes)
openfaas-opennsfw-5bcb55bb77-x9vhf   128m         28Mi
```
I did not see an example to borrow from  in store & functions, or the stack.yml of inception or the workshop, so am not sure what to do next.   Any thoughts much appreciated.

**Pull request merged**